src/DataStructures/DataVector.hpp

```diff
@@ -58,8 +58,12 @@ using std::abs;  // NOLINT
 // IWYU pragma: no_include <blaze/math/traits/MultTrait.h>
 // IWYU pragma: no_include <blaze/math/traits/SubTrait.h>
 // IWYU pragma: no_include <blaze/system/TransposeFlag.h>
-// IWYU pragma: no_include <blaze/math/traits/BinaryMapTrait.h>
+#if ((BLAZE_MAJOR_VERSION == 3) && (BLAZE_MINOR_VERSION <= 3))
```

Comment:
Maybe the more compact thing to do would be to add type aliases `blaze_enable_if` etc., use those and `ifdef` there?